src/devices/HuskyLens/Point.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Iot.Device.HuskyLens
+{
+    /// <summary>
+    /// Represents apoint in camera coordinates
+    /// </summary>
+    public class Point
```

Comment:
We do already have points in System.Drawing. And it covers this scenario and more. Any specific reason to have this one?
You add in the comment some rage but you do not seem to test them, so really wondering why not using the already existing class.

Reply:
See: https://docs.microsoft.com/en-us/dotnet/api/system.drawing?view=dotnet-plat-ext-3.1#remarks

> The System.Drawing namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service and it is not cross-platform. ImageSharp and SkiaSharp are recommended as alternatives.

This class seemed lightweight enough to include here instead of introducting a dependency on a drawing framework, just to be able to express a tuple of ints. maybe a tuple is better here?

Reply:
@krwq any opinion on this?

Reply:
for now please use System.Drawing to be consistent with other places and please file an issue on that. If we change it then let's do it globally

Reply:
Also if you're fine with floats you can also use Vector2